conf/modules.config

```diff
@@ -18,24 +18,152 @@ process {
+        ]
+        ext.args = "-x splice:hq -uf --secondary=no -a"
+    }
+
+    withName: GUNZIP {
+        publishDir = [
+            path: { "${params.outdir}/06.1_GUNZIP" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = { "${meta.id}_tama" }
+    }
+
+    withName: ULTRA_PIPELINE {
+        publishDir = [
+            path: { "${params.outdir}/06.2_ULTRA" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args = "--isoseq --disable_infer"
+    }
+
+    withName: PERL_BIOPERL {
+        publishDir = [
+            path: { "${params.outdir}/06.3_PERL_BIOPERL" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args = '-F\'\\t\' -nae \'print if ($F[5] !~ /N$/)\''
+        ext.prefix = { "${meta.id}_filtered.sam" }
+    }
+
+    withName: SAMTOOLS_SORT {
+        publishDir = [
+            path: { "${params.outdir}/07_SAMTOOLS_SORT" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.prefix = { "${meta.id}_sorted" }
     }
 
-    withName: CUSTOM_DUMPSOFTWAREVERSIONS {
+    if (params.capped == true) {
+        withName: GSTAMA_COLLAPSE {
+            publishDir = [
+                path: { "${params.outdir}/08_GSTAMA_COLLAPSE" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+            ext.args = "-x capped -b BAM -a ${params.five_prime} -m ${params.splice_junction} -z ${params.three_prime}"
+        }
+    } else {
+        withName: GSTAMA_COLLAPSE {
+            publishDir = [
+                path: { "${params.outdir}/08_GSTAMA_COLLAPSE" },
+                mode: 'copy',
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+            ext.args = "-x no_cap -b BAM -a ${params.five_prime} -m ${params.splice_junction} -z ${params.three_prime}"
+        }
+    }
+
+    withName: GSTAMA_FILELIST {
         publishDir = [
-            path: { "${params.outdir}/pipeline_info" },
-            mode: params.publish_dir_mode,
-            pattern: '*_versions.yml'
+            path: { "${params.outdir}/09_GSTAMA_FILELIST" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
 
+    withName: GSTAMA_MERGE {
+        publishDir = [
+            path: { "${params.outdir}/10_GSTAMA_MERGE" },
+            mode: 'copy',
+            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+        ]
+        ext.args = "-a ${params.five_prime} -m ${params.splice_junction} -z ${params.three_prime}"
+    }
+
+//  withName: SAMPLESHEET_CHECK {
```

Comment:
`SAMPLESHEET_CHECK` and `CUSTOM_DUMPSOFTWAREVERSIONS` shouldn't be commented, should they?

Reply:
Fixed